 index b560e74..f4c0d55 10064
 --- a/vc1/dcs.yaml
 +++ b/vc1/dcs.yaml
 @@ -1,5 +1,5 @@
  seaf.ta.reverse.vmwareonprem.vdcs:
 -  vdcs.datacenter-3:
+  mangerok.gora.vdcs.datacenter-3:
     id: datacenter-3
     original_id: vim.Datacenter:datacenter-3
     title: PP-Datacenter
@@ -32,31 +32,34 @@ seaf.ta.reverse.vmwareonprem.vdcs:
     - vim.Datastore:datastore-55429
     - vim.Datastore:datastore-55430
     networks:
-    - network.network-52456
-    - network.network-55558
-    - network.network-55563
-    - network.network-55559
-    - network.network-62296
-    - network.network-56532
-    - network.network-56533
-    - network.network-56530
-    - network.network-56531
-    - network.dvportgroup-16451
-    - network.dvportgroup-42578
-    - network.dvportgroup-1008
-    - network.dvportgroup-35
-    - network.dvportgroup-1015
-    - network.dvportgroup-33
-    - network.dvportgroup-1019
-    - network.dvportgroup-6247
-    - network.dvportgroup-2006
-    - network.dvportgroup-39093
-    - network.dvportgroup-2007
-    - network.dvportgroup-42577
-    - network.dvportgroup-1009
-    - network.dvportgroup-25991
-    - network.dvportgroup-34
-    - network.dvportgroup-1012
-    - network.dvportgroup-8336
-    - network.dvportgroup-2022
-    dc: ''
+    - mangerok.gora.network.network-52456
+    - mangerok.gora.network.network-55558
+    - mangerok.gora.network.network-55563
+    - mangerok.gora.network.network-55559
+    - mangerok.gora.network.network-62296
+    - mangerok.gora.network.network-76099
+    - mangerok.gora.network.network-56532
+    - mangerok.gora.network.network-76110
+    - mangerok.gora.network.network-56533
+    - mangerok.gora.network.network-56530
+    - mangerok.gora.network.network-56531
+    - mangerok.gora.network.dvportgroup-75222
+    - mangerok.gora.network.dvportgroup-16451
+    - mangerok.gora.network.dvportgroup-42578
+    - mangerok.gora.network.dvportgroup-1008
+    - mangerok.gora.network.dvportgroup-35
+    - mangerok.gora.network.dvportgroup-1015
+    - mangerok.gora.network.dvportgroup-33
+    - mangerok.gora.network.dvportgroup-1019
+    - mangerok.gora.network.dvportgroup-6247
+    - mangerok.gora.network.dvportgroup-2006
+    - mangerok.gora.network.dvportgroup-39093
+    - mangerok.gora.network.dvportgroup-2007
+    - mangerok.gora.network.dvportgroup-42577
+    - mangerok.gora.network.dvportgroup-1009
+    - mangerok.gora.network.dvportgroup-25991
+    - mangerok.gora.network.dvportgroup-34
+    - mangerok.gora.network.dvportgroup-1012
+    - mangerok.gora.network.dvportgroup-8336
+    - mangerok.gora.network.dvportgroup-2022
+    dc: mglk.dc.gora
diff --git a/vc1/dvportgroups_datacenter-3.yaml b/vc1/dvportgroups_datacenter-3.yaml
index 3201b90..70a0609 100644
--- a/vc1/dvportgroups_datacenter-3.yaml
+++ b/vc1/dvportgroups_datacenter-3.yaml
@@ -1,199 +1,210 @@
 seaf.ta.reverse.vmwareonprem.dvportgroups:
-  dvportgroup.dvportgroup-8336:
+  mangerok.gora.dvportgroup.dvportgroup-8336:
     id: dvportgroup-8336
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-8336
     title: 602_SSID-Guest
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '602'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-1009:
+  mangerok.gora.dvportgroup.dvportgroup-1009:
     id: dvportgroup-1009
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-1009
     title: 505_DMZ
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '505'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-2006:
+  mangerok.gora.dvportgroup.dvportgroup-2006:
     id: dvportgroup-2006
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-2006
     title: 514_SOIB-SERVERS-WNAM
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '514'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-6247:
+  mangerok.gora.dvportgroup.dvportgroup-6247:
     id: dvportgroup-6247
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-6247
     title: 517_Servers_KSB
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '517'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-33:
+  mangerok.gora.dvportgroup.dvportgroup-33:
     id: dvportgroup-33
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-33
     title: DSwitch-DVUplinks-32
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - 0 - 4094
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-39093:
+  mangerok.gora.dvportgroup.dvportgroup-75222:
+    id: dvportgroup-75222
+    original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-75222
+    title: 520_DHCP-SSID-Guest
+    description: ''
+    subnets: ''
+    dvswitch: mangerok.gora.dvswitch.dvs-32
+    vlan:
+    - '520'
+    vdc: mangerok.gora.vdc.datacenter-3
+    vdc_title: PP-Datacenter
+  mangerok.gora.dvportgroup.dvportgroup-39093:
     id: dvportgroup-39093
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-39093
     title: 616_TestVLAN
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '616'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-25991:
+  mangerok.gora.dvportgroup.dvportgroup-25991:
     id: dvportgroup-25991
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-25991
     title: 503_dManagement-EPH
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '503'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-1012:
+  mangerok.gora.dvportgroup.dvportgroup-1012:
     id: dvportgroup-1012
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-1012
     title: 600_WiFi_AP
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '600'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-2022:
+  mangerok.gora.dvportgroup.dvportgroup-2022:
     id: dvportgroup-2022
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-2022
     title: 648_ASDU
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '648'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-1019:
+  mangerok.gora.dvportgroup.dvportgroup-1019:
     id: dvportgroup-1019
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-1019
     title: 504_MONITORING
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '504'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-34:
+  mangerok.gora.dvportgroup.dvportgroup-34:
     id: dvportgroup-34
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-34
     title: 503_dManagement
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '503'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-42577:
+  mangerok.gora.dvportgroup.dvportgroup-42577:
     id: dvportgroup-42577
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-42577
     title: 508_dManagement
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '508'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-1015:
+  mangerok.gora.dvportgroup.dvportgroup-1015:
     id: dvportgroup-1015
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-1015
     title: 630_Metra+AXXESS
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '630'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-35:
+  mangerok.gora.dvportgroup.dvportgroup-35:
     id: dvportgroup-35
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-35
     title: 512_SERVERS
     description: ''
     subnets: ''
-    dvswitch: dvswitch.dvs-32
+    dvswitch: mangerok.gora.dvswitch.dvs-32
     vlan:
     - '512'
-    vdc: vdc.datacenter-3
+    vdc: mangerok.gora.vdc.datacenter-3
     vdc_title: PP-Datacenter
-  dvportgroup.dvportgroup-1008:
+  mangerok.gora.dvportgroup.dvportgroup-1008:
     id: dvportgroup-1008
     original_id: vim.dvs.DistributedVirtualPortgroup:dvportgroup-1008
     title: 502_IP-PBX